src/App.js

```diff
@@ -51,10 +51,10 @@ export class App extends Component {
       ],
       infoText: [
         "",
-        "You see a unusually long and heavy sword lying next to you.",
-        "As you pick it up, you notice from the corner of your eye a wounded boar charging at you.",
+        "You see an unusually long and heavy sword lying next to you.",
+        "As you pick it up, you notice a wounded boar charging at you from the corner of your eye.",
```

Comment:
wouldn't "from **the** corner of the eye" be even better ?

Reply:
Looks the same to me, but you're the boss :wink: